sites/cnfpt/src/backend/base/admin.py

```diff
@@ -39,3 +42,5 @@ def ajax_upload(request, folder_id=None):
 user_model = auth.get_user_model()
 admin.site.unregister(user_model)
 admin.site.register(user_model, CustomUserAdmin)
+
+cms.cache.page.set_page_cache = set_page_cache
```

Comment:
I think we need a comment to explain that we are monkey patching and why with a link to the problem in DjangoCMS code.

Reply:
I added a comment to explain the monkey patch.